rules-reviewed/technology-usage/tests/database-technology-usage.windup.test.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" ?>
+<ruletest id="technology-usage-database-test" xmlns="http://windup.jboss.org/schema/jboss-ruleset">
+    <testDataPath>data/database/</testDataPath>
+    <rulePath>../database-technology-usage.windup.xml</rulePath>
+    <ruleset>
+        <rules>
+            <rule id="technology-usage-database-01000-test">
```

Comment:
See comment below for test `technology-usage-database-01300-test`

Reply:
?  I don't get it? There is data source in `mysql-ds.xml` file. We don't identify these? We identify only data sources in `persistence.xml` ?